theorems/T000057.md

```diff
@@ -1,14 +1,12 @@
 ---
 uid: T000057
 if:
-  P000121: true
+  P000144: true
 then:
   P000028: true
 refs:
-- doi: 10.1007/978-1-4612-6290-9
-  name: Counterexamples in Topology
+  - mathse: 4659902
+    name: How can the implication metrizable -> first countable be generalized?
 ---
 
-If $X$ is pseudometric, for any $x \in X$, $\{B(x,\frac{1}{n})\,|\,n \in \omega\}$ is a countable local base at $x$.
-
-Asserted on page 34-35 of {{doi:10.1007/978-1-4612-6290-9}} for metric spaces.  The same holds for pseudometric spaces.
+Take a pseduometrizable neighborhood. The balls of rational radius within this neighborhood form a local basis.
```

Comment:
I didn't catch this before accepting the push request, but "pseudometrizable" is misspelled; it appears as "pseduometrizable"

Reply:
I'll make a hotfix for this directly on main.

Reply:
> I'll make a hotfix for this directly on main.

Seems T327 has the same missplelling.

Reply:
Fixed now in 70e67cd. I'm okay if maintainers/reviewers do trivial (literally one line) typo changes directly on main.

Go to e.g. https://github.com/pi-base/data/blob/main/theorems/T000327.md click the pencil icon to get a quick editor, make the fix, change the commit message to say "fixed typo" (I forgot to do this at 70e67cd), then commit directly to main.

Reply:
Not sure it would let me do this.  Just to check, I clicked on the pencil icon and at the bottom it says "you can't commit to main, because it's a protected branch. Create a new branch for this commit and start a pull request."

Reply:
Oops. I guess that might be an admin-only feature. So in that case I still recommend the workflow when convenient, but rather than committing to main, create a "patch" branch and open a PR that can be quickly reviewed/accepted by others.